packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -1,38 +1,33 @@
-import type { Connector } from 'wagmi';
-import type { Chain } from '../components/RainbowKitProvider/RainbowKitChainContext';
-import type { WalletList } from './Wallet';
+import { CreateConnectorFn } from 'wagmi';
```

Comment:
Is `import type` still viable here? Seeing a few places where we should use that (like the `Chain` type)